src/run.php

```diff
@@ -29,7 +29,11 @@
 
     $app = new HiveApplication($config, $logger, $dataFolder);
     if ($app['action'] !== 'run') {
-        $app['logger']->setHandlers([new NullHandler(Logger::INFO)]);
+        // On sync actions -> log only errors
```

Comment:
Pridal som tu to iste, co sme riesili pre `extractor`: https://github.com/keboola/db-extractor-hive/pull/6
 ... aby pouzivatel vedel preco nepreslo `testConnection`.